apps/src/gamelab/GameLab.js

```diff
@@ -429,7 +434,7 @@ GameLab.prototype.reset = function (ignore) {
   }
 };
 
-GameLab.prototype.onPuzzleComplete = function (submit) {
+GameLab.prototype.onPuzzleComplete = function () {
```

Comment:
We'll need to add this back at some point (when we make submittable GameLab levels work), but right now this was never being called with submit=true, and so I decided to remove.